build.yaml

```diff
@@ -1699,6 +1699,17 @@ libs:
   filegroups:
   - grpc_test_util_base
   secure: true
+- name: engine_passthrough
+  build: test
+  language: c
+  src:
+  - test/core/end2end/engine_passthrough.cc
+  dll: only
+  platforms:
```

Comment:
`dll` and `platforms` don't seem required here.

Reply:
dll: only is required as engine are not static libraries. They are dynamic. 

I have removed platforms

Reply:
Let me try removing dll and will run the tests.

Reply:
No we can't remove dll: only as without that it produces .a file which is a static library and cannot be used as openssl engine. It has to be dynamic. Hence, we require dll: only.

Reply:
I see. Thanks.